tests/unit/s2n_renegotiate_test.c

```diff
@@ -407,7 +407,7 @@ int main(int argc, char *argv[])
      */
     {
         /* Setup a security policy that only contains one cipher */
-        struct s2n_cipher_preferences one_cipher_preference = { .count = 1, .suites = NULL };
+        struct s2n_cipher_preferences one_cipher_preference = { .count = 1, .suites = NULL, .allow_chacha20_boosting = false};
```

Comment:
Did this actually fail? Updating every test is unnecessary. 0==false, so your new field will be initialized to false for every existing cipher preferences.

Reply:
No, this is more of an after-affect of a different bug I encountered before. I can remove it